workflow/controller/operator.go

```diff
@@ -87,6 +93,13 @@ type wfOperationCtx struct {
 	// preExecutionNodePhases contains the phases of all the nodes before the current operation. Necessary to infer
 	// changes in phase for metric emission
 	preExecutionNodePhases map[string]wfv1.NodePhase
+
+	// wfSpec holds the WorkflowSpec for execution
+	wfSpec *wfv1.WorkflowSpec
+	// entrypoint is the starting point for workflow execution
+	entrypoint string
```

Comment:
This field doesn't seem to be actually used anywhere. Why is this needed? If it isn't, can you remove it?

Reply:
Good catch